### PR TITLE
Expose the enableIPv6 setting

### PR DESCRIPTION
**Pull request by aidanhs**
docker/docker#17513

### CODE REVIEW

File: network.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ type NetworkInfo interface {
 	IpamInfo() ([]*IpamInfo, []*IpamInfo)
 	DriverOptions() map[string]string
 	Scope() string
+	EnableIPv6() bool
```
Comment (aboch) [inline network.go]: Thanks for the change.
Can we rename this to `IPv6Enabled()`. 
`EnableIPv6()` sounds more like an command, rather than a query.

Comment (aidanhs) [inline network.go]: I was on the fence - I wanted to strive for consistency (e.g. Internal returns internal and it's called EnableIPv6 throughout the change in docker/docker), but it does sound like a command name. I've made the change as requested.

**Comment by aboch** [comment]
LGTM

**Comment by mrjana** [comment]
LGTM for this change.

But @aidanhs if you are working on adding `--ipv6` to docker network create command then you probably need an api change in libnetwork to pass that down here.

**Comment by aidanhs** [comment]
@mrjana happily not, the work to pick up enableipv6 has already been done for me - https://github.com/docker/libnetwork/blob/master/network.go#L477

**Comment by mrjana** [comment]
Ah cool. Thanks

**Pull request merged**